docs/user-guide.md

```diff
@@ -0,0 +1,856 @@
+    expensive and blocking computation can run somewhere other than the main
+    application thread that controls the user interface. When such work is done
+    on the main thread, the browser appears frozen. Web workers ensure
+    expensive blocking computation happens elsewhere. Think of workers as
+    independent subprocesses in your web page.</dd>
+
+    <dt><em>Rich and powerful plugins</em></dt>
+    <dd>
+    <p>As you'll see, PyScript has a small, efficient yet powerful core called
+    <a href="https://github.com/pyscript/polyscript">PolyScript</a>.</p>
+    <p>Most of the functionality of PyScript is actually implemented through
+    PolyScript's <a href="#plugins">plugin system</a>.</p>
+
+    <p>This approach means we get a clear separation of concerns: PolyScript
+    can focus on being small, efficient and powerful, whereas the PyScript
+    related plugins allow us to build upon the generic features provided by
+    PolyScript.</p>
+    <p>More importantly, because there is a plugin system, folks
+    <em>independent of the PyScript core team</em> have a way to create their
+    own plugins so we get a rich ecosystem of functionality that reflects the
+    unique and diverse needs of PyScript's users.</p>
+    </dd>
+</dl>
+
+## First steps
+
+It's simple:
+
+* tell your browser to use PyScript, then,
+* tell PyScript how to run your Python code.
+
+For the browser to use PyScript, simply add a `<script>` tag, whose `src`
+attribute references a CDN url for `pyscript.core`, to your HTML document's
+`<head>`. You may also add a reference to optional PyScript related CSS:
+
+**TODO: USE CORRECT CDN URL**:
+
+```html title="Reference PyScript in your HTML"
+<!doctype html>
+<html>
+    <head>
+        <!-- Recommended meta tags -->
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <!-- optional PyScript CSS -->
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@pyscript/core/dist/core.css">
+        <!-- This script tag bootstraps PyScript -->
+        <script type="module" src="https://cdn.jsdelivr.net/npm/@pyscript/core"></script>
+    </head>
+    <body>
+        <!-- your code goes here... -->
+    </body>
+</html>
+```
+
+There are two ways to tell PyScript how to find your code.
+
+* With a standard HTML `<script>` tag whose `type` attribute is either `py`
+  (for Pyodide) or `mpy` (for MicroPython). **This is currently the
+  recommended way**.
+* Via the bespoke `<py-script>` (Pyodide) and `<mpy-script>` (MicroPython)
+  tags. Historically, this used to be the only way to reference your code.
+
+These should be inserted into the `<body>` of your HTML document.
+
+In both cases either use the `src` attribute to reference a Python
+file containing your code, or inline your code between the opening and closing
+tags. **We recommend you use the `src` attribute method**, but retain the
```

Comment:
Yes we do. You're editing a Python file, rather than editing an HTML with Python in it (without the code highlighting). I'm sure you and I have talked about this many times before.